Common/Export as Image in ASP.NET Core 6.0/ExportImageindotnet6/ExportImageindotnet6/ExportImageindotnet6.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Syncfusion.EJ2.AspNet.Core" Version="20.4.0.44" />
-    <PackageReference Include="Syncfusion.EJ2.PdfViewer.AspNet.Core.Windows" Version="20.4.0.44" />
+    <PackageReference Include="Syncfusion.EJ2.AspNet.Core" Version="23.1.41" />
+    <PackageReference Include="Syncfusion.EJ2.PdfViewer.AspNet.Core.Windows" Version="23.1.41" />

```

Comment:
@SaravanaPriya31 - Can you use the common package in the sample?

Reply:
resolved